lib/statem_dsl.ex

```diff
@@ -151,469 +155,340 @@ defmodule PropCheck.StateM.DSL do
       property "run the sequential cache", [:verbose] do
         forall cmds <- commands(__MODULE__) do
           Cache.start_link(@cache_size)
-          execution = run_commands(cmds)
+          {_history, _state, result} = run_commands(cmds)
           Cache.stop()
-          (execution.result == :ok)
+          (result == :ok)
         end
       end
 
   ## Increasing the Number of Commands in a Sequence
+
   Sometimes issues can hide when the command sequences are short. In order to
   tease out these hidden bugs we can increase the number of commands generated
   by using the `max_size` option in our property.
 
         property "run the sequential cache", [max_size: 250] do
         forall cmds <- commands(__MODULE__) do
           Cache.start_link(@cache_size)
-          execution = run_commands(cmds)
+          {_history, _state, result} = run_commands(cmds)
           Cache.stop()
-          (execution.result == :ok)
+          (result == :ok)
         end
   """
 
-  use PropCheck
-  alias PropCheck.BasicTypes
-  import PropCheck.Logger, only: [log_error: 1]
-
-  @typedoc """
-  The name of a command must be an atom.
-  """
-  @type command_name :: atom
-  @typedoc """
-  A symbolic state can be anything and appears only during phase 1.
-  """
+  @type symb_var :: :proper_statem.symbolic_var()
```

Comment:
Please add the type documentation again!

Reply:
Since this was taken straight from StateM and the issue applies to both modules, so I'll address it in a separate PR to fix both of them.